deepspeed/checkpoint/universal_checkpoint.py

```diff
@@ -103,6 +114,55 @@ def load_hp_checkpoint_state(self, folder, tp_rank, tp_world_size):
 
             hp_mapping.optim_fragment[key] = tp_hp_fragment.clone().detach()
 
+    return step
+
+
+def load_hp_checkpoint_state_from_checkpoint_dir(zero_optimizer: ZeROOptimizer, lp_groups_name: str,
```

Comment:
Should we make this a method of `ZeROOptimizer` class?

Reply:
Thanks @tohtana.

Reply:
@tjruwase Yeah, I have missed your comment, sorry. It was good that the merge failed yesterday.